lib/ingram_micro.rb

```diff
@@ -15,4 +16,8 @@ def self.configure
     yield configuration
     configuration.assert_valid
   end
+
+  def self.generate_order_number(prefix='')
```

Comment:
@dl103 @sent-hil that's a good point, we should enforce arg is an integer or string of an integer. We should also seek some clarity from IM about the requirements for that field.

To your second point @sent-hil, I agree that we need more/better documentation. I'll add comments, maybe utilizing TomDoc.